### PR TITLE
Add feerate_percentiles to blockinfo endpoint

### PR DESCRIPTION
**Pull request by benbuschmann**
Add feerate_percentiles to blockinfo

### CODE REVIEW

**Comment by raphjaph** [comment]
Why only the 10th percentile? We could consider allowing all of them. 

Just out of interest, what are you building with this?

**Comment by benbuschmann** [comment]
> Why only the 10th percentile? We could consider allowing all of them.
> 
> Just out of interest, what are you building with this?

I am looking for the lowest fee that was included in a block and min_fee_rate seems unreliable. I will be using the low fee rate to trigger a dynamic inscription.  I can add all 5 percentiles later today if you think it's worth doing all at once.

**Comment by raphjaph** [comment]
As part of the strict backwards compatibility requirements for recursive endpoints we can only add fields, never remove or change an existing one. So I think adding all of them into one field makes sense here.

**Comment by benbuschmann** [comment]
@raphjaph ok I added feerate_percentiles, does this look correct to you?

File: docs/src/inscriptions/recursion.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Examples
   "confirmations": 0,
   "difficulty": 0.0,
   "hash": "000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f",
+  "feerate_percentiles": [0, 0, 0, 0, 0],
```
[the remark this answers is not in the document]
Comment (benbuschmann) [reply docs/src/inscriptions/recursion.md:90]: just added a note

File: docs/src/inscriptions/recursion.md [hunk 2]
```diff
@@ -76,6 +76,8 @@ Examples
 
 - `/r/blockinfo/0`:
 
+Note: `feerate_percentiles` are feerates at the 10th, 25th, 50th, 75th, and 90th percentile weight unit (in satoshis per virtual byte).
```
Comment (raphjaph) [inline docs/src/inscriptions/recursion.md:79]:
```suggestion
Note: `feerate_percentiles` are feerates at the 10th, 25th, 50th, 75th, and 90th percentile in sat/vB).
```

**Review by raphjaph — CHANGES_REQUESTED** [review]
Looks good, just a small nit

**Review by raphjaph — APPROVED** [review]
LGTM

**Pull request merged**